crates/router/src/types/storage/refund.rs

```diff
@@ -69,4 +95,76 @@ impl RefundDbExt for Refund {
             .change_context(errors::DatabaseError::NotFound)
             .attach_printable_lazy(|| "Error filtering records by predicate")
     }
+
+    async fn filter_by_meta_constraints(
+        conn: &PgPooledConn,
+        merchant_id: &str,
+        refund_list_details: &api_models::refunds::RefundListRequest,
+        limit: i64,
+        offset: i64,
```

Comment:
This shouldn't have a limit/offset concept since it's only metadata...
If the user specified a limit 1 he means that on the number of rows. He still wants to know all the currency/connector/status that exists...